src/scripts/main.js

```diff
@@ -1,3 +1,37 @@
 'use strict';
 
-// write your code here
+const successMessage = () => {
+  const message = document.createElement('div');
+
+  message.className = 'message';
+  message.innerText = 'Promise was resolved!';
+
+  return document.body.append(message);
+};
+
+const errorMessage = () => {
+  const message = document.createElement('div');
+
+  message.className = 'message error-message';
+  message.innerText = 'Promise was rejected!';
+
+  return document.body.append(message);
+};
+
+const promise1 = new Promise((resolve) => {
+  const logo = document.querySelector('.logo');
+
+  logo.addEventListener('click', () => {
+    resolve();
+  });
+});
+
+const promise2 = new Promise((resolve, reject) => {
+  setTimeout(() => {
+    reject(new Error());
```

Comment:
The `promise2` rejection is correctly implemented with a timeout, but consider providing a more descriptive error message or handling the error in a way that gives more context about the rejection.